cypress/support/config/services.js

```diff
@@ -1495,7 +1495,9 @@ const genServices = appEnv => ({
       frontPage: { path: '/yoruba', smoke: false },
       liveRadio: { path: undefined, smoke: false },
       mediaAssetPage: {
-        path: isLive(appEnv) ? undefined : '/yoruba/media-23256797',
+        path: isLive(appEnv)
+          ? '/yoruba/afrika-51116686'
+          : '/yoruba/media-23256797', // live is video clip
```

Comment:
The first link is what will run in live - the bottom link will run everywhere else

Just concerned this might not be intentional given the comment :)

Reply:
Oh yeah sorry, the position of the comment makes it unclear

Reply:
The first one is the live URL I intended :)